src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/CustomTagEntryExtensionsTests.cs

```diff
@@ -78,7 +78,12 @@ public void GivenStandardTagAsKeyword_WhenFormalizing_ThenVRShouldBeFilled()
         public static IEnumerable<object[]> GetValidCustomTagEntries()
         {
             yield return new object[] { DicomTag.DeviceSerialNumber.BuildCustomTagEntry() }; // standard custom tag with VR
-            yield return new object[] { new CustomTagEntry("12051003", DicomVRCode.OB, CustomTagLevel.Instance, CustomTagStatus.Added) }; // private tag with VR
+            yield return new object[] { CreateCustomTagEntry("12051003", DicomVRCode.OB, CustomTagLevel.Instance, CustomTagStatus.Added) }; // private tag with VR
+        }
+
+        private static CustomTagEntry CreateCustomTagEntry(string path, string vr, CustomTagLevel level = CustomTagLevel.Instance, CustomTagStatus status = CustomTagStatus.Added)
+        {
+            return new CustomTagEntry() { Path = path, VR = vr, Level = level, Status = status };
```

Comment:
nit: remove the `()` for the ctor

Reply:
never know this. :)

Reply:
A bit of a shortcut lol. It is only valid for this object initializer syntax